java/src/jmri/jmrit/display/layoutEditor/LayoutEditor.java

```diff
@@ -124,42 +124,6 @@
     static final ResourceBundle rbx = ResourceBundle.getBundle("jmri.jmrit.display.DisplayBundle");
     static final ResourceBundle rbean = ResourceBundle.getBundle("jmri.NamedBeanBundle");
 
-    // size of point boxes
-    private static final double SIZE = 3.0;
-    private static final double SIZE2 = SIZE * 2.;  // must be twice SIZE
-
-    // connection types
-    final public static int NONE = 0;
```

Comment:
I am talking about *every* _final public static int_ field that you are removing from this class from ```NONE``` to ```TURNTABLE_RAY_OFFSET```.

See @pabender's comment about why.